notes-webapp/src/main/resources/locale/portlet/Portlets_en.properties

```diff
@@ -0,0 +1,9 @@
+#############################################################################
+#SpaceApplications
+#############################################################################
+
+SpaceSettings.application.wiki.title=Notes
+SpaceSettings.application.wiki.description=Wiki Portlet
```

Comment:
The old wiki app is supposed to have been removed why are we keeping this ? 

CC @mkrout